diff -Npur blender-2.62.orig/CMakeLists.txt blender-2.62/CMakeLists.txt
--- blender-2.62.orig/CMakeLists.txt	2012-02-17 14:46:00.705398679 +0100
+++ blender-2.62/CMakeLists.txt	2012-02-17 14:49:05.631395413 +0100
@@ -157,7 +157,7 @@ endif()
 if(UNIX AND NOT APPLE)
 	option(WITH_X11_XINPUT "Enable X11 Xinput (tablet support and unicode input)"	ON)
 	option(WITH_X11_XF86VMODE "Enable X11 video mode switching"	ON)
-	option(WITH_BUILTIN_GLEW "Use GLEW OpenGL wrapper library bundled with blender" ON)
+	option(WITH_BUILTIN_GLEW "Use GLEW OpenGL wrapper library bundled with blender" OFF)
 	option(WITH_XDG_USER_DIRS    "Build with XDG Base Directory Specification (only config and documents for now)" OFF)
 	mark_as_advanced(WITH_XDG_USER_DIRS)
 else()
diff -Npur blender-2.62.orig/extern/CMakeLists.txt blender-2.62/extern/CMakeLists.txt
--- blender-2.62.orig/extern/CMakeLists.txt	2012-02-17 14:46:00.617398680 +0100
+++ blender-2.62/extern/CMakeLists.txt	2012-02-17 14:48:31.043396011 +0100
@@ -40,10 +40,6 @@ if(WITH_BINRELOC)
 	add_subdirectory(binreloc)
 endif()
 
-if(WITH_BUILTIN_GLEW)
-	add_subdirectory(glew)
-endif()
-
 if(WITH_GAMEENGINE)
 	add_subdirectory(recastnavigation)
 endif()
diff -Npur blender-2.62.orig/extern/SConscript blender-2.62/extern/SConscript
--- blender-2.62.orig/extern/SConscript	2012-02-17 14:46:00.612398681 +0100
+++ blender-2.62/extern/SConscript	2012-02-17 14:47:40.178396919 +0100
@@ -2,7 +2,6 @@
 
 Import('env')
 
-SConscript(['glew/SConscript'])
 SConscript(['colamd/SConscript'])
 
 if env['WITH_BF_GAMEENGINE']:
diff -Npur blender-2.62.orig/source/blender/editors/include/BIF_gl.h blender-2.62/source/blender/editors/include/BIF_gl.h
--- blender-2.62.orig/source/blender/editors/include/BIF_gl.h	2012-02-17 14:46:00.843398676 +0100
+++ blender-2.62/source/blender/editors/include/BIF_gl.h	2012-02-17 14:48:07.365396454 +0100
@@ -33,7 +33,7 @@
 #ifndef BIF_GL_H
 #define BIF_GL_H
 
-#include "GL/glew.h"
+#include <GL/glew.h>
 
 	/*
 	 * these should be phased out. cpack should be replaced in
