--- extern/SConscript
+++ extern/SConscript
@@ -2,8 +2,6 @@
 
 Import('env')
 
-SConscript(['glew/SConscript'])
-
 if env['WITH_BF_GAMEENGINE']:
	 SConscript(['recastnavigation/SConscript'])

 if env['WITH_BF_ELTOPO']:
	 SConscript(['eltopo/SConscript'])

--- source/blender/editors/include/BIF_gl.h
+++ source/blender/editors/include/BIF_gl.h
@@ -35,7 +35,7 @@
 #ifndef BIF_GL_H
 #define BIF_GL_H
 
-#include "GL/glew.h"
+#include <GL/glew.h>
 
	/*
	 * these should be phased out. cpack should be replaced in
